source/adapters/offload/program.cpp

```diff
@@ -31,15 +32,17 @@ ur_result_t ProgramCreateCudaWorkaround(ur_context_handle_t hContext,
   cuLinkComplete(State, &CuBin, &CuBinSize);
   RealBinary = (uint8_t *)CuBin;
   RealLength = CuBinSize;
+
+#if 0
```

Comment:
Should this still be here?

Reply:
I intentionally left it as a debugging thing that can be enabled if necessary